edx_proctoring/templates/proctored_exam/visit_exam_content.html

```diff
@@ -37,4 +37,10 @@
       }
     );
   </script>
+{% elif has_due_date %}
+  <p>
+    {% blocktrans %}
+      Once the due date for this exam has passed, you will be able to go visit this page and review your answers.
```

Comment:
"After the due date for this exam has passed, you will be able to review your answers on this page."